server/documents/elements/header.html.eco

```diff
@@ -256,6 +256,35 @@ themes      : ['Default', 'Classic', 'Bookish', 'Chubby', 'Material']
     </h3>
   </div>
 
+  <div class="example" data-since="2.9.4">
```

Comment:
Please add the dedicated classnames for automativ highlighting
```suggestion
  <div class="example" data-since="2.9.4" data-class="seamless,attached">
```